tests/CMakeLists.txt

```diff
@@ -1402,13 +1402,15 @@ foreach(prec IN LISTS TEST_PRECS)
       --enable-testing true
       --gtest_output=xml:staggered_eigensolve_test_staggered_${prec}.xml)
 
-    add_test(NAME eigensolve_test_asqtad_${prec}
-      COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:staggered_eigensolve_test> ${MPIEXEC_POSTFLAGS}
-      --dslash-type asqtad --compute-fat-long true
-      --eig-n-conv 32 --eig-n-ev 32 --eig-n-kr 256
-      --dim 6 6 6 8 --prec ${prec} --eig-tol ${tol} --eig-max-restarts 1000
-      --enable-testing true
-      --gtest_output=xml:staggered_eigensolve_test_staggered_${prec}.xml)
+    # Skipping this because it's both time consuming and doesn't have any novel spectral
```

Comment:
Wondering if we should make this an additional CMake option to allow for programmatic enablement on demand for whatever reason?

Reply:
Eh... it feels like we're going to start running down a rabbit hole if we do that---if we have a flag specifically for twisted mass, why don't we have one for every other Dirac operator, etc, etc...

Reply:
....this is asqtad not twisted mass, but sure.

Reply:
Oh yes. Sorry, I looked very quickly and my brain segfaulted.

I agree with you there---I can add that, that's a good suggestion.

Reply:
https://github.com/lattice/quda/pull/1421/commits/4e533feb810b3f3ba27c42729a8059c5c0b286a1